test/ApplicationInsightsJavaScriptSnippetTest/Rendered.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page - ApplicationInsightsJavaScriptSnippetSample</title>
+    <script>document.write('Head tag script. Look for Application Insights JavaScriptSnippet below this script in the HTML file.')</script>
+    <script type="text/javascript">
```

Comment:
I actually wonder how stable this is... I have a feeling this test will break the moment that App Insights changes the JS that is used. Can we come up with a way to make the test less fragile? Perhaps just making sure that "some kind of script tag" got rendered where we wanted it to get rendered? Not sure what's a good way to do that, but I want to avoid a potentially fragile test.